flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -7,6 +7,9 @@
     "org.flathub.exceptions_wildcard": {
         "*": "ignore all errors and warnings"
     },
+    "page.codeberg.libre_menu_editor.LibreMenuEditor": {
+        "finish-args-unnecessary-xdg-data-access": "app-id predates this linter rule"
```

Comment:
please write an actually reason instead if senselessly cut and pasting incorrect information.

Reply:
This information is not incorrect. I wrote and submitted this app before these new linter-rules existed. I just want to push an update to my app